examples/analog_hamiltonian_simulation/06_Solving_SAT_with_Differential_programming.ipynb

```diff
@@ -0,0 +1,872 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                                           wires=range(8))
</pre><p>How about <code>wires=range(len(coords))</code> for added customizability? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>